### DISCUSSION TITLE
Size of reference file?

### DISCUSSION THREAD

**Discussion by danchurch** [LooseLab/readfish#220] [General]
Hi!

I read your very exciting paper, and I'm happy to see that you are keeping your software maintained for the benefit of the rest of us. 

But forgive a very ignorant/obvious question: what is your recommended size limit on reference genomes? 

From the title of your paper ("Readfish enables targeted nanopore sequencing of gigabase-sized genomes") and the human cell line you used,  I infer human size genomes should be fine. But I can't find any further explanations on the issue of size limits.

I'm hoping to use a large, complex, repetitive conifer genome (picea abies, 20 gbases!) as my reference genome, to deplete host reads. I'm still not sure this is currently possible, but your package looks the most promising so far. 

Best,
Daniel

**Comment by mattloose** [comment]
Good question!

This may be somewhat system dependent - you will obvioulsy need enough memory available to load your reference - then the efficiency of the whole process will depend on the speed of the mapping. We are soon (early next year) to release a slightly faster version of minimap2 - this might help a little too.

To test this, I would probably do some benchmarking of how fast you can map on your system to this reference. Hopefully we will have some examples of how to do this in the new year,

Matt

**Comment by danchurch** [comment]
Great, thanks! I will keep an eye on the repo. -dan

**Comment by danchurch** [comment]
I wanted to ask, do you think that removing repetitive regions of the genome would improve mapping speeds with minimap2? I don't know if this would just be redundant with the indexing algorithms of minimap, or if it might yield some improvements here.
